sif/job-companion

```diff
@@ -15,7 +15,8 @@ Stage: builder
 	cd /usr/local/go/src/github.com/sylabs/slurm-operator
 	CGO_ENABLED=0 go build -o main cmd/job-companion/main.go
 
-Bootstrap: scratch
+Bootstrap: busybox
```

Comment:
Why build a busybox container from scratch instead of using one from the library?

Reply:
no, thanks for suggestion, updated!